gateway/gateway.go

```diff
@@ -304,9 +304,9 @@ func (gateway *HandleT) dbWriterWorkerProcess(process int) {
 		}
 
 		if gwAllowPartialWriteWithErrors {
-			errorMessagesMap = gateway.jobsDB.StoreWithRetryEach(jobList)
+			errorMessagesMap = gateway.jobsDB.AppendWithRetryEach(jobList)
 		} else {
-			err := gateway.jobsDB.Store(jobList)
+			err := gateway.jobsDB.Append(jobList)
```

Comment:
Some alternative names
`Append` -> `Push`, `Publish`, `Enqueue`

Reply:
**Poll**

👍  Append
👎  Push
😄  Publish
🎉  Enqueue
❤️  Store

Reply:
.Go - Tells jobslist to go in jobsdb 😁

Reply:
I voted to keep `Store` although it is more generic as a name just to avoid confusion because inside it is processing db requests with a specific name and that name has not changed. It is still invoked as `store` [here](https://github.com/rudderlabs/rudder-server/blob/b5a90cfe67405e47db73557b6b997ec01c91d03c/jobsdb/jobsdb.go#L3742) and that is used for metrics.

I prefer to have a closer match between code and metrics, I would find it less confusing. If that is not the case please let me know.

Reply:
Will retain the old name since renaming would require changes in the metrics, grafana dashboards and it is not the main focus of this pull request